tests/lib/__init__.py

```diff
@@ -486,6 +508,10 @@ def main():
             entry_points=dict(console_scripts=['{name}={name}:main'])
         )
     """.format(name=name)))
+    return _vcs_add(script, version_pkg_path, vcs)
```

Comment:
@xavfernandez thanks for the review. I don't understand this comment. I simply split the existing _create_tests_package method in two, extracting the _vcs_add part in order to reuse it in _create_test_package_with_srcdir.